omero/developers/build-system.txt

```diff
@@ -5,157 +5,79 @@ Build System
-      |   |-- log4j.xml
+      |   |-- logback.xml
       |   |-- omero.properties
       |   \-- profiles
       |
-      |-- examples ............................ User examples
+      |-- examples ........................... User examples
       |
       \components
         |
+        |--<component-name> .................. Each component has this same basic structure.
+        |    |-- build.xml ................... Build file
+        |    |-- ivy.xml ..................... Jar dependencies
+        |    |-- test.xml .................... Test dependencies
+        |    |-- src ......................... Source code
+        |    |-- resources ................... Other files of interest
+        |    |-- test ........................ Test source code and test resources
+        |    \-- target ...................... Build output (deleted on clean)
         |
-        |--<component-name> ................... Each component has this same basic structure.
-        |    |-- build.xml                      Main scripts
-        |    |-- ivy.xml                        Jar dependencies
-        |    |-- test.xml                       Test dependencies
-        |    |-- src                            Source code
-        |    |-- resources                      Other files of interest
-        |    |-- test                           Test source code and test resources
-        |    \-- target                         Output of build (deleted on clean)
+        |  NOTABLE COMPONENTS
         |
-        |-- model ............................. The model component is special in that in produces
-        |                                       a jar specific to your database choice: model-psql.jar
-        |                                       The generated `ome.model.*` files contain Hibernate
-        |                                       annotations for object-relational mapping.
+        |-- model ............................ The model component is special in that it produces
+        |                                      a jar specific to your database choice: model-psql.jar
+        |                                      The generated `ome.model.*` files contain Hibernate
+        |                                      annotations for object-relational mapping.
         |
-        |-- blitz ............................. The blitz component also performs code generation
-        |    |                                  producing artifacts for Java, Python, and C++.
-        |    |
-        |    \ blitz_tools.py ................. OMERO-specific SCons Environment definition
-        |                                       and other build tools.
+        |-- blitz ............................ The blitz component also performs code generation
+        |    |                                 producing artifacts for Java, Python, and C++.
+        |    \ blitz_tools.py ................ OMERO-specific SCons Environment definition
```

Comment:
Given the above chnage from SCons to Cmake is the mention of SCons here still relevant?